docs/topics/mta-cli-analyze.adoc

```diff
@@ -73,9 +116,8 @@ dependency.log
 output.yaml
 static-report
 ----
++
+* `output.yaml` is the file that contains the issues report.
+* `static-report` contains the static HTML report.
+* `dependencies.yaml` contains a dependencies report.
```

Comment:
```suggestion
* `dependencies.yaml` contains the dependencies report.
```

This is pedantic but as you use `contains the * report` twice previously, you could use the same pattern in the third bullet